docs/sql/system.txt

```diff
@@ -688,6 +688,34 @@ _node
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+recovery
+--------
+
++------------------------------------------+----------------------------------------------------+-------------+
+| Column Name                              | Description                                        | Return Type |
++==========================================+====================================================+=============+
+| ``recovery``                             | Represents recovery statistic of the particular    | ``Object``  |
+|                                          | index.                                             |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']``                     | Statistics about physical index recovery in bytes. | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['total_bytes']``      | Total number of bytes in the index.                | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['reused_bytes']``     | Number of reused bytes for the index recovery.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['recovered_bytes']``  | Number of actual bytes recovered in the index.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']``                    | Statistics about physical index recovery in files. | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['total']``           | Total number of files in the index.                | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['reused']``          | Number of reused files for the index recovery.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['recovered']``       | Number of actual files recovered in the index.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['total_time']``               | Total time to recover the index in ms.             | ``Long``    |
```

Comment:
Is this an estimate on how long it is expected to take for the recovery to finish or is this the amount of time the recovery has taken so far?